handlers/node_api.py

```diff
@@ -60,18 +54,201 @@ def get_address_search(
 
     response = node_api.get_address_search(address, count, page, hash, token)
 
-    if response is None:
-        return {
-            "statusCode": 503,
-            "body": json.dumps({"error": "unknown_error"}),
-            "headers": {
-                "Content-Type": "application/json"
-            }
+    return {
+        "statusCode": 200,
+        "body": json.dumps(response or UNKNOWN_ERROR_MSG),
+        "headers": {
+            "Content-Type": "application/json"
+        }
+    }
+
+
+@ApiGateway()
+def get_version(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    node_api: Optional[NodeApi] = None
+) -> dict:
+    node_api = node_api or NodeApi()
+
+    response = node_api.get_version()
+
+    return {
+        "statusCode": 200,
+        "body": json.dumps(response or UNKNOWN_ERROR_MSG),
+        "headers": {
+            "Content-Type": "application/json"
+        }
+    }
+
+
+@ApiGateway()
+def get_dashboard_tx(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    node_api: Optional[NodeApi] = None
+) -> dict:
+    node_api = node_api or NodeApi()
+
+    block = event.query.get("block")
+    tx = event.query.get("tx")
```

Comment:
Maybe we should add warnings here that any new query parameter that someone eventually adds here will have to be added to serverless.yml's cacheKeys as well.

It seems very easy to make this kind of mistake.

Reply:
Done on the docstrings